test/e2e/flask/simulation-preview.mocks.ts

```diff
@@ -0,0 +1,90 @@
+import type { Mockttp } from "mockttp";
+
+const MOCK_SENDER_ADDRESS = '0x5cfe73b6021e818b776b421b1c4db2474086a7e1';
```

Comment:
Super minor, but is it standard for us to use `MOCK` as the suffix so all the variables don't start with `MOCK`?